src/GovUk.Education.ExploreEducationStatistics.Admin/Areas/Identity/Pages/Account/ExternalLogin.cshtml.cs

```diff
@@ -104,21 +104,34 @@ public async Task<IActionResult> OnGetCallbackAsync(string returnUrl = null, str
             {
                 var nameClaim = info.Principal.FindFirstValue(JwtClaimTypes.Name);
 
-                if (nameClaim.IndexOf(' ') > 0)
+                if (nameClaim == null)
```

Comment:
In future we could look at splitting this up a bit, maybe in another service, as `OnGetCallbackAsync` has reached 140+ lines. 🙂